### PR TITLE
Fix "Profile Update Not Working on Admin Portal #1508"

### LINKED ISSUE #1508: Profile Update Not Working on Admin Portal
**Describe the bug**
Any Changes to The Admin Profile cannot be made . We get a 404 error in response.

**To Reproduce**

1.Log in Admin Panel as Admin
2.Go to Update Profile section
3.Change Profile Details
4.Click Update Profile

**Expected behavior**
Profile Information Should be Updated

**Actual behavior**
Response is error

**Screenshots**
[Screencast from 28-01-24 04:42:58 PM IST.webm](https://github.com/PalisadoesFoundation/talawa-admin/assets/121512095/9d263248-15f8-4964-be7f-2b9f36166795)

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Shubh152**
**What kind of change does this PR introduce?**

bugfix

**Issue Number:**

Fixes #1508 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/121512095/ec9f17e7-21f6-4484-965d-9658d14f2fe4

**If relevant, did you update the documentation?**

No

**Summary**

https://github.com/PalisadoesFoundation/talawa-admin/issues/1508

**Does this PR introduce a breaking change?**

**Other information**

Another PR needs to be merged in talawa-api repo to fix the issue completely

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@Shubh152 Please fix the test application error.

**Comment by Shubh152** [comment]
The error is not generated due to changes in this PR. I think this is a general problem faced in other PRs as well. I have not made any changes that might have created this error. Please look into it.

**Comment by Cioppolo14** [comment]
@rishav-jha-mech @anwersayeed Can you review this PR?

**Comment by rishav-jha-mech** [comment]
@Shubh152 add screenshots/videos of the flow that you have made

**Comment by Shubh152** [comment]
I have also made a relevant PR in the talawa-api repo

**Comment by Shubh152** [comment]
made review changes as requested in the api repo PR

**Comment by palisadoes** [comment]
_**NOTE**_ Read very carefully

1. We just merged this PR which upgraded the prettier package. 
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/1599
1. It reformatted over 150 files. 

This will put your PR at risk of extensive merge conflicts. Do the following **IN THIS ORDER**:

1. upgrade your `prettier` in your local branch to the same version
1. run `prettier` on your local branch
1. update your local branch with the latest upstream
    
This will help to reduce the number of **existing** and **future** merge conflicts for your PR.

**Comment by Shubh152** [comment]
Please review the PR. It only introduces a Basic UI change

File: src/components/UserUpdate/UserUpdate.test.tsx [hunk 1]
```diff
@@ -100,31 +122,34 @@ describe('Testing User Update', () => {
...
+      formData.firstName
     );
     userEvent.type(
       screen.getByPlaceholderText(/Last Name/i),
-      formData.lastName,
+      formData.lastName
     );
     userEvent.type(screen.getByPlaceholderText(/Email/i), formData.email);
-    userEvent.selectOptions(screen.getByTestId('applangcode'), 'Français');
+    // userEvent.selectOptions(screen.getByTestId('applangcode'), 'Français');
```
Comment (rishav-jha-mech) [inline src/components/UserUpdate/UserUpdate.test.tsx:139]: Why have you commented this line in tests

**Comment by rishav-jha-mech** [comment]
@Shubh152 don't push any commented test, also please revert any change you have made inside talawa-admin-docs folder

File: src/components/UserUpdate/UserUpdate.test.tsx (hunk 1 above)
Comment (Shubh152) [reply src/components/UserUpdate/UserUpdate.test.tsx]: Sorry that was not intended.

**Comment by Cioppolo14** [comment]
@Shubh152 Please fix the codecov failing tests.

**Comment by palisadoes** [comment]
1. Please fix the failing tests
2. Try to get the code coverage for the patch closer to that of the repository

**Comment by palisadoes** [comment]
Please get the code coverage of your patch closer to that of the repository

**Comment by noman2002** [comment]
@Shubh152 Its been more than 3 weeks for the PR, please get this merged.

**Comment by Shubh152** [comment]
Will be done this week. I was caught up with exams last week

**Comment by Cioppolo14** [comment]
@Shubh152 How is this going?

**Comment by pranshugupta54** [comment]
Hey @Cioppolo14, I think this will be automatically fixed in #1667 with the new design

**Comment by Shubh152** [comment]
Stuck in a RTL testing error. I have reverted some changes as proper testing was not working.

**Comment by palisadoes** [comment]
1. This PR needs to be closed. The source and target PR branches are the same. This means that our reviewers will not be able to checkout the changes and test on their systems using this methodology which will cause a rejection error:
    1. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally
1. Please create a new branch in your local repository with the changes
2. Make sure the branch name is named differently from the target branch (`develop`)
3. Resubmit your PR

You will see that other PRs are using non-develop branch names for this reason

**Pull request closed without merging**